docs/how_to/label_annotator.md

```diff
@@ -0,0 +1,33 @@
+# Paint labels on arbitrary planes
+
+## Summary
+
+Painting labels is useful for annotating voxels by semantic meaning or instance. Painting labels on planes in 3D is useful to gain full context. Additionally, the axis aligned slicing may not be optimal for viewing your structure of interest. Here we describe how to paint labels on arbitrary planes in 3D.
+
+![type:video](https://github.com/napari-threedee/napari-threedee/assets/1120672/6750b262-6c79-425f-8af5-d7459ba28a16)
+
+## Instructions
+
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/label_annotator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/label_annotator_plugin.py`. 
```

Comment:
```suggestion
1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/label_annotator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable as you will have to run the script. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/label_annotator_plugin.py`. 
```

Reply:
I'm going to dismiss this and update to the same wording as used in the other PR(s).